gitops/azure-devops/unit_test/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Hidden files generated by macOS.
+.DS_Store
```

Comment:
@yradsmikham the top level gitignore file doesn't exclude these?

Reply:
It is. I can remove the .gitignore. It was previously in a separate working directory and was brought in from there.